eng/pipelines/templates/stages/cosmos-sdk-client.yml

```diff
@@ -34,6 +34,18 @@ stages:
               OSVmImage: "windows-2019"
               NodeTestVersion: "8.x"
               TestType: "node"
+            Windows_Node10:
```

Comment:
Do you want to run these all on Windows for some reason, or should we spread the node versions across Linux and Mac as well, which I believe we do for most other pipelines?

Reply:
This issue shows the matrix we use for most JS tests:

https://github.com/Azure/azure-sdk-for-js/issues/9462

For "live" tests (which I believe are the same as "integration" tests mentioned in this issue), the matrix is:

**Test Type**|**OS**|**Node**|**Cloud**
-----|-----|-----|-----
Live|Windows|8| 
Live|Windows|Browser (Edge)| 
Live|Linux|10| 
Live|Linux|12| 
Live|Linux|Browser (Firefox)| 
Live|MacOS|14| 
Live|MacOS|Browser (Safari)|

Reply:
We also have https://github.com/Azure/azure-sdk/issues/1650 which is about getting cosmos to use the standard templates we use for everything else in the repo. That would prevent us from needing to maintain this matrix separately.

Reply:
I believe the reason for running only on Windows is that Cosmos Emulator works only on Windows machine as per Steve Faulkner.

Reply:
Correct

Reply:
@mikeharder I would love to run it across our normal matrix, but the problem is this:

https://github.com/Azure/azure-sdk-for-js/blob/master/eng/common/pipelines/templates/steps/cosmos-emulator.yml

The cosmos emulator is Windows only right now and the VMs aren't able to run it on any other platform.

Reply:
Do we run tests against live cosmos service on all OS?

Reply:
> Do we run tests against live cosmos service on all OS?

The unit tests run across the normal matrix, but I don't know about live tests. @southpolesteve ?

Reply:
We no longer have any live tests

Reply:
Thanks for clarifying.